src/daemon/main.cpp

```diff
@@ -233,6 +247,13 @@ int main(int argc, char const * argv[])
     boost::filesystem::path data_dir = boost::filesystem::absolute(
         command_line::get_arg(vm, cryptonote::arg_data_dir));
 
+#ifdef WIN32
+    if (isFat32(data_dir.root_name().c_str()))
+    {
+      MERROR("Data directory resides on FAT32 volume that has 4GiB file size limit, blockchain might get corrupted.");
+    }
```

Comment:
+ return error ?

Reply:
Think we shouldn't. We didn't hit a error yet (using `MERROR` to print the red warning). Will work pretty fine till the blockchain grows to 4GiB.
Who knows what the actual blockchain size is. Also

Also will prevent running the daemon in `--no-sync` mode.